tools/loader/ImageClassifier.cpp

```diff
@@ -135,11 +157,17 @@ int main(int argc, char **argv) {
 
   // Load and process the image data into the data Tensor.
   Tensor data;
-  loadImagesAndPreprocess(inputImageFilenames, &data, imageMode);
+
+  loadImagesAndPreprocess(inputImageFilenames, &data, imageMode, loadimageFormat);
 
   // All of our ONNX and Caffe2 models may use one of two names for input.
   constexpr char const *c2Inputs[2] = {"gpu_0/data", "data"};
-  constexpr char const *onnxInputs[2] = {"gpu_0/data_0", "data_0"};
+  
+  char const *onnxInputs[2] = {"gpu_0/data_0", "data_0"};
```

Comment:
The only reason we have it set up like this is the models we have run so far all have two possible input names. Does the input model you are running actually have two different input variable name possibilities too? It might not be necessary for other models, and we perhaps should refactor this.

Reply:
I am not sure for now, the models I have currently evaluated have only one name. So I would like to keep it this way for now, as I validate more models, I will keep this in mind.